### ISSUE TITLE
Female gender selection

### ISSUE THREAD

**Issue by lare96** [lare96/asteria-2.0#101]
For some reason the female gender either crashes the client or causes the player's name and model to appear weird, I'm not sure as of yet as if this is only through the character selection screen or if the player cannot be a female character at all.

An example here:
http://www.rune-server.org/runescape-development/rs2-server/help/580685-asteria-2-0-female-problem.html

**Comment by lare96** [comment]
I can't seem to find an issue for this server-sided. To clarify the issue a bit, it seems as if the character can only be a female when armour is equipped; for instance, these screenshots were taken when I modified the appearance update block to FORCE the player to appear as a female.

Here is without armor, the player looks completely male even though I am sending the data for it to be female.

![screenshot_1](https://cloud.githubusercontent.com/assets/3312404/6205669/c13a4e12-b544-11e4-9ba6-af7b12256661.png)


Then here is the same player, after I've equipped armour. As you can see, the player looks like a female (as in the body is less bulk, etc.).

![screenshot_2](https://cloud.githubusercontent.com/assets/3312404/6205678/de58faa2-b544-11e4-91b1-757a0b459bff.png)

**Comment by lare96** [comment]
Finally got this bug fixed! What was causing it had nothing to do with player updating, but it was the character selection screen packet being decoded incorrectly. The reason why the "armor" bug I commented about above was happening was because I was sending data to appear as a female while having male appearance models.

The character selection screen has been fixed, and here is the result...

![](http://i.imgur.com/MKz1RWX.png)

The fix can be found here https://github.com/lare96/asteria-3.0/commit/c81d6a75db8884e975331ba1bb0aef1a67eccd19
